assets/ios/NotificationService.m

```diff
@@ -135,7 +142,64 @@ - (void)didReceiveNotificationRequest:(UNNotificationRequest *)request withConte
         self.bestAttemptContent.body = [NSString stringWithFormat:@"You received %.0f sats ⚡️", amountInSats];
     }
 
-    self.contentHandler(self.bestAttemptContent);
+    if ([notificationType isEqualToString:@"payment_received"] && ttsEnabled) {
+        NSString *speechText         = [NSString stringWithFormat:@"%.0f sats", amountInSats];
```

Comment:
can the code in this if be moved to a new method?

Reply:
Done